src/MAUI/Maui.Samples/Samples/Layers/AddCustomDynamicEntityDataSource/AddCustomDynamicEntityDataSource.xaml.cs

```diff
@@ -25,8 +26,12 @@ namespace ArcGIS.Samples.AddCustomDynamicEntityDataSource
         instructions: "Run the sample to view the map and the dynamic entity layer displaying the latest observation from the custom data source. Tap on a dynamic entity to view its attributes in a callout.",
         tags: new[] { "data", "dynamic", "entity", "label", "labeling", "live", "real-time", "stream", "track" })]
     [ArcGIS.Samples.Shared.Attributes.ClassFile("SimulatedDataSource.cs")]
+    [ArcGIS.Samples.Shared.Attributes.OfflineData("a8a942c228af4fac96baa78ad60f511f")]
     public partial class AddCustomDynamicEntityDataSource
     {
+        // Path to AIS Traffic Data json file.
```

Comment:
Your choice: "Vessel AIS Traffic Data" would more closely match name of online item and could be slightly clearer.